src/window/dxgi/src/lib.rs

```diff
@@ -84,11 +73,17 @@ where Cf: format::RenderFormat
 /// Initialize with a given size. Raw format version.
 pub fn init_raw(title: &str, requested_width: winapi::INT, requested_height: winapi::INT, color_format: format::Format)
                 -> Result<(Window, Device, Factory, core::handle::RawRenderTargetView<Resources>), InitError> {
-    let hwnd = match window::create(title, requested_width, requested_height) {
-        Ok(h) => h,
-        Err(()) => return Err(InitError::Window),
+    let inner = match winit::WindowBuilder::new()
+        .with_title(title.to_owned())
+        .with_dimensions(requested_width as u32, requested_height as u32)
+        .with_min_dimensions(requested_width as u32, requested_height as u32)
```

Comment:
hmm, I see the reasoning. We'll revise this later then, when the resize support is better.